exercises/isogram/isogram_test.rb

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'isogram'
+
+# Test data version:
+# 306975e
+class IsogramTest < Minitest::Test
+  def test_duplicates
+    str = 'duplicates'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_eleven
+    skip
+    str = 'eleven'
+    refute Isogram.is_isogram?(str)
+  end
+
+  def test_subdermatoglyphic
+    skip
+    str = 'subdermatoglyphic'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_alphabet
+    skip
+    str = 'Alphabet'
+    refute Isogram.is_isogram?(str)
+  end
+
+  def test_thumbscrew_japingly
+    skip
+    str = 'thumbscrew-japingly'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_hjelmqvist_gryb_zock_pfund_wax
```

Comment:
I'd prefer to see more meaningful test names that indicated what the test was for.

eg: `test_long_string_with_multiple_hyphens`

(I know you're getting the data from the common json file, so this might not be possible in the time you have available, but it might be nice to contribute a pull request which added a `description` to each test that could be used to generate meaningful names to the https://github.com/exercism/x-common/blob/master/isogram.json)

You should also try to ensure that failing tests have meaningful failure messages, see issue #386